src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +28,27 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn str-to-int [s]
+  #?(:clj (try
+            (when s
+              (java.lang.Integer/parseInt s))
+            (catch NumberFormatException e
+              s))
+     :cljs (let [x (js/parseInt s)]
+             (when-not (js/isNaN x)
+               s))))
```

Comment:
You could call rems.common.util/parse-int here, and perhaps also rename this function (since it doesn't always return an int now). How about maybe-parse-int or something like that?

Reply:
sounds good! Will do!